Example/RZTouchIDDemo/RZLoggedInViewController.h

```diff
@@ -0,0 +1,13 @@
+//
+//  RZLoggedInViewController.h
+//  RZTouchIDDemo
+//
+//  Created by Adam Howitt on 1/9/15.
+//  Copyright (c) 2015 Raizlabs. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
```

Comment:
`@import UIKit;` for demo project files, but I think public source headers should not use modules.